courseraresearchexports/commands/containers.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraresearchexports.containers import client
+from courseraresearchexports import utils
+from datetime import datetime
+
+
+def create_container(args):
+    """
+    Create a container containing a postgres database using an export job id.
+     Export job will be downloaded and loaded into dockerized databse.
+     Automatically starts container.
+    """
+    try:
+        d = utils.docker_client(args)
+        container = client.create_from_export_job_id(
+            export_job_id=args.exportJobId, docker_client=d)
+
+        print('Successfully created container {}'.format(container['Id']))
+
+    except:
+        print('Error creating container.')
+
+
+def list_containers(args):
+    """
+    List docker containers with Coursera research exports
+    """
+    d = utils.docker_client(args)
+    containers = client.list_all(docker_client=d)
+    print('Name\tCreated\tStatus'.expandtabs(18))
```

Comment:
I would suggest using a library to generate tables to display to the user, for example, https://pypi.python.org/pypi/tabulate